docs/topics/yaml-rule-structure-syntax.adoc

```diff
@@ -361,17 +361,28 @@ This provider has the following capabilities:
 
 .`referenced`
 
-The `referenced` capability enables the provider to find references in the source code. This capability takes two input parameters, `pattern` and `location`.
+The `referenced` capability enables the provider to find references in the source code. This capability takes three input parameters, `pattern`, `location`, and 'annotated'.
 
 [source,terminal]
 ----
 when:
   java.referenced:
     pattern: "<pattern>" <1>
     location: "<location>" <2>
+    annotated: "<annotated>" <3>
 ----
 <1> A regular expression pattern to match, for example, `org.kubernetes.*`.
 <2> Specifies the exact location where the pattern needs to be matched, for example, `IMPORT`.
+<3> Checks for specific annotations and their elements, such as name and value in the Java code using a query. For example, the following query matches the Bean(url = “http://www.example.com”) annotation in the method.
```

Comment:
```suggestion
<3> Checks for specific annotations and their elements, such as name and value in the Java code using a query. For example, the following query matches the Bean (url = “http://www.example.com”) annotation in the method.
```

Reply:
This one appears without a space in the code, so I guess it should be fine.